profiles/package.mask

```diff
@@ -1,3 +1,3 @@
 # André Erdmann <dywi@mailerd.de> (15 Sep 2014)
 # ebuild builds against two live repos
->=app-laptop/tlp-9999
+app-laptop/tlp
```

Comment:
This would mask *all* TLP ebuilds.

The ``>=`` is necessary here: only live ebuilds should be masked, not the regular ones (``1.1``, ...).